packages/react-native-bpk-component-navigation-bar/src/BpkNavigationBarBackButtonIOS.ios.js

```diff
@@ -53,7 +54,7 @@ const styles = StyleSheet.create({
 
 const BpkNavigationBarBackButtonIOS = (props: Props) => {
   const { title, showTitle, onPress, tintColor } = props;
-  const tintColorFinal = tintColor || colorSkyGray;
+  const tintColorFinal = tintColor || useBpkDynamicValue(textPrimaryColor);
```

Comment:
Hooks should not be used with conditionals, can we move the call to the hook a line up and just use the value here?